README.md

```diff
@@ -445,11 +445,15 @@ matrix(
 Prettier ships with a handful of customizable format options, usable in both the CLI and API.
 
 ### Print Width
-Specify the length of line that the printer will wrap on.
-
-**We strongly recommend against using more than 80 columns.**
-
-Prettier works by cramming as much content as possible until it reaches the limit, which happens to work well for 80 columns but makes lines that are very crowded. When a bigger column count is used in styleguides, it usually means that code is allowed to go beyond 80 columns, but not to make every single line go there, like Prettier would do.
+Specify the line length that the printer will wrap on.
+
+> **For readability we recommend against using more than 80 characters:**
+>
+>In code styleguides, maximum line length rules are often set to 100 or 120.
+>
+>This is not the amount of characters that will always be on a line. For readability a developer will use whitespace to break up long lines so the average amount of characters will be 80 characters on a line.
+>
+>Prettier will put the maximum amount of characters on a line. With the print width set to 120 every line would have around 110 - 120 characters. This is harder to read than the recommended 80 characters.
```

Comment:
Boy, I may be falling into the clarity trap that is trying to be clarified here.

Is this actually trying to say that a `--print-width 110` will make lines that contain 110 _non-whitespace characters_, resulting in lines that potentially have more than 110 columns?

I'm now not sure if `--print-width` determines the number of columns on a line or the number of non-whitespace characters on a line.

Apologies if I've raised an invalid point.  Any clarification would be much appreciated.